Common/src/main/java/dk/sdu/se4/common/entity/parts/MovingPart.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package dk.sdu.se4.common.entity.parts;
+
+import dk.sdu.se4.common.entity.Entity;
+import java.awt.Point;
+
+/**
+ *
+ * @author steff
+ */
+public class MovingPart implements EntityPart {
+
+    private Point deltaPoint;
+    private float speed;
+    private boolean UP, DOWN, RIGTH, LEFT;
+
+    public MovingPart(float speed) {
+        this.speed = speed;
+
+    }
+
+    public Point getDeltaPoint() {
+        return deltaPoint;
+    }
+
+    public void setDeltaPoint(Point deltaPoint) {
```

Comment:
what does delta point mean?

Reply:
it the point from the whare to calculate the point

Reply:
I don't use point. locate I use translate 
I will fix that